lib/domain/board.rb

```diff
@@ -5,5 +5,42 @@ class Game
   def initialize(board_width:)
     size = board_width * board_width
     @board = Array.new(size)
+    @winning_combinations = [
```

Comment:
Is this necessary? I assume the `find_winning_combinations` use case does this already.

https://github.com/C-gyorfi/tic-tac-toe/blob/24af5772e02f80f05b595c651c828afd55cd6aa1/lib/domain/board.rb#L8-L17

Reply:
There is also inconsistency with file name and class name.

Reply:
Fixed